packages/babel-types/src/definitions/core.js

```diff
@@ -276,6 +279,15 @@ defineType("File", {
     program: {
       validate: assertNodeType("Program"),
     },
+    comments: {
+      validate: assertEach(assertNodeType("Comment")),
+      optional: true,
+    },
```

Comment:
Just FYI this seems to have broken Jests `toMatchInlineSnapshot` matcher for tests written in TypeScript (https://github.com/facebook/jest/issues/10208)

Reply:
good catch, this is not intended, effect… probably this should be considered a breaking change and so to go to babel 8